documentation/modules/adding-source-provider.adoc

```diff
@@ -83,34 +83,53 @@ endif::[]
 
 .Procedure
 
-. In the {ocp} web console, click *Migration* -> *Providers for virtualization*.
-. Click *Create Provider*.
 ifdef::vmware[]
-. Click *vSphere*.
-. Specify the following fields:
+. Access the *Create provider* page for {vmw} by doing one of the following:
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project is `openshift-mtv`. If you have Administrator privileges, you can see all projects, otherwise, you can see only the projects you are authorized to work with.
```

Comment:
```suggestion
... Select a *Project* from the list. 
The default project is `openshift-mtv` if active project is set to 'All projects' or the active project otherwise. 
If you have Administrator privileges, you can see all projects, otherwise, you can see only the projects you are authorized to work with.
```

Reply:
@RichardHoch Confirmed.

The current/active project is almost always shown on page header:

![Screenshot from 2025-07-03 14-18-16](https://github.com/user-attachments/assets/5849a57c-23f6-4f8f-b2b3-3fa0eab48267)

I would replace the "the one shown on the Providers page"  to maybe "current/active project shown in page header." I think this term of "active project" or "current project" is already used in the mtv doc....